com.hazelcast.jet.core.metrics.JobExecutionMetricsTest.testExecutionMetricsBatchJob [HZ-1858]

_master_ (commit 2e152898593c1c9d0960524b1d43b118df4739ba)

Failed on RedHatJDK8: https://jenkins.hazelcast.com/job/Hazelcast-master-RedHatJDK8/67/testReport/junit/com.hazelcast.jet.core.metrics/JobExecutionMetricsTest/testExecutionMetricsBatchJob/
<details><summary>Stacktrace:</summary>

```
java.lang.AssertionError: Expected a value of at least 1669946065100 for metric 'executionCompletionTime', but got 1669946065099
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at com.hazelcast.jet.core.metrics.JobMetricsChecker.assertAggregatedMetricValueAtLeast(JobMetricsChecker.java:97)
	at com.hazelcast.jet.core.metrics.JobMetricsChecker.assertSummedMetricValueAtLeast(JobMetricsChecker.java:64)
	at com.hazelcast.jet.core.metrics.JobExecutionMetricsTest.testExecutionMetricsBatchJob(JobExecutionMetricsTest.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

```
</details>


<details><summary>Standard output:</summary>

```
Finished Running Test: testExecutionMetricsStreamJob in 0.468 seconds.
Started Running Test: testExecutionMetricsBatchJob
01:54:25,051 DEBUG || - [JobClassLoaderService] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Creating job classLoader for job 0908-6189-1108-0001
01:54:25,051 DEBUG || - [JobClassLoaderService] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Create processor classloader map for job 0908-6189-1108-0001
01:54:25,057 DEBUG || - [Planner] hz.compassionate_brown.cached.thread-3 - Watermarks in the pipeline will be throttled to 1000
01:54:25,062  INFO || - [JobCoordinationService] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Starting job 0908-6189-1108-0001 based on submit request
01:54:25,067  INFO || - [MasterJobContext] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Didn't find any snapshot to restore for job '0908-6189-1108-0001', execution 0908-6189-1109-0001
01:54:25,068  INFO || - [MasterJobContext] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Start executing job '0908-6189-1108-0001', execution 0908-6189-1109-0001, execution graph in DOT format:
digraph DAG {
	"items" [localParallelism=1];
	"loggerSink" [localParallelism=1];
	"items" -> "loggerSink" [queueSize=1024];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
01:54:25,068 DEBUG || - [MasterJobContext] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Building execution plan for job '0908-6189-1108-0001', execution 0908-6189-1109-0001
01:54:25,069 DEBUG || - [MasterJobContext] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Built execution plans for job '0908-6189-1108-0001', execution 0908-6189-1109-0001
01:54:25,070 DEBUG || - [InitExecutionOperation] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Initializing execution plan for job 0908-6189-1108-0001, execution 0908-6189-1109-0001 from [127.0.0.1]:5701
01:54:25,095  INFO || - [JobExecutionService] hz.compassionate_brown.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Execution plan for jobId=0908-6189-1108-0001, jobName='0908-6189-1108-0001', executionId=0908-6189-1109-0001 initialized
01:54:25,096 DEBUG || - [MasterJobContext] hz.compassionate_brown.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Init of job '0908-6189-1108-0001', execution 0908-6189-1109-0001 was successful
01:54:25,096 DEBUG || - [MasterJobContext] hz.compassionate_brown.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Executing job '0908-6189-1108-0001', execution 0908-6189-1109-0001
01:54:25,097  INFO || - [JobExecutionService] hz.compassionate_brown.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Start execution of job '0908-6189-1108-0001', execution 0908-6189-1109-0001 from coordinator [127.0.0.1]:5701
01:54:25,099  INFO || - [WriteLoggerP] hz.compassionate_brown.jet.blocking.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] [0908-6189-1108-0001/loggerSink#0] 0
01:54:25,099  INFO || - [WriteLoggerP] hz.compassionate_brown.jet.blocking.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] [0908-6189-1108-0001/loggerSink#0] 1
01:54:25,099  INFO || - [WriteLoggerP] hz.compassionate_brown.jet.blocking.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] [0908-6189-1108-0001/loggerSink#0] 2
01:54:25,099  INFO || - [WriteLoggerP] hz.compassionate_brown.jet.blocking.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] [0908-6189-1108-0001/loggerSink#0] 3
01:54:25,099  INFO || - [WriteLoggerP] hz.compassionate_brown.jet.blocking.thread-0 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] [0908-6189-1108-0001/loggerSink#0] 4
bbb: [metric=classloading.totalLoadedClassesCount,excludedTargets={}], v=48134
bbb: [thread=hz.compassionate_brown.priority-generic-operation.thread-0,unit=count,metric=operation.thread.completedRunnableCount,excludedTargets={}], v=0
bbb: [unit=ns,metric=partitions.elapsedDestinationCommitTime,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.genericQueueSize,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.priority-generic-operation.thread-0,unit=count,metric=operation.thread.completedPacketCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.priority-generic-operation.thread-0,unit=count,metric=operation.thread.completedOperationBatchCount,excludedTargets={}], v=0
bbb: [unit=count,metric=partitions.replicaSyncSemaphore,excludedTargets={}], v=10
bbb: [unit=count,metric=operation.genericThreadCount,excludedTargets={}], v=3
bbb: [unit=count,metric=operation.completedCount,excludedTargets={}], v=722
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.completedOperationCount,excludedTargets={}], v=213
bbb: [partitionId=6,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=71
bbb: [unit=count,metric=operation.adhoc.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=166
bbb: [metric=classloading.unloadedClassesCount,excludedTargets={}], v=433
bbb: [unit=count,metric=operation.runningPartitionCount,excludedTargets={}], v=0
bbb: [metric=os.committedVirtualMemorySize,excludedTargets={}], v=49304469504
bbb: [unit=ns,metric=partitions.elapsedMigrationTime,excludedTargets={}], v=0
bbb: [unit=count,metric=partitions.migrationQueueSize,excludedTargets={}], v=0
bbb: [unit=count,metric=event.threadCount,excludedTargets={}], v=1
bbb: [unit=count,metric=jobs.submitted,module=jet,excludedTargets={}], v=4
bbb: [metric=runtime.usedMemory,excludedTargets={}], v=1308835592
bbb: [unit=ns,metric=partitions.totalElapsedMigrationTime,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.maxHeap,excludedTargets={}], v=8478261248
bbb: [unit=ms,metric=cluster.clock.maxClusterTimeDiff,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.completedOperationCount,excludedTargets={}], v=343
bbb: [cooperativeWorker=1,unit=count,metric=iterationCount,module=jet,excludedTargets={}], v=5198
bbb: [unit=ns,metric=partitions.totalElapsedMigrationOperationTime,excludedTargets={}], v=0
bbb: [metric=os.processCpuTime,excludedTargets={}], v=2385340000000
bbb: [thread=hz.compassionate_brown.generic-operation.thread-1,unit=count,metric=operation.thread.completedOperationCount,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.committedHeap,excludedTargets={}], v=8478261248
bbb: [unit=count,metric=raft.terminatedRaftNodeGroupIds,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.freeHeap,excludedTargets={}], v=7169425656
bbb: [partitionId=5,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=53
bbb: [metric=os.freePhysicalMemorySize,excludedTargets={}], v=143138627584
bbb: [unit=count,metric=event.totalFailureCount,excludedTargets={}], v=0
bbb: [unit=count,metric=partitions.totalCompletedMigrations,excludedTargets={}], v=0
bbb: [cooperativeWorker=0,unit=count,metric=iterationCount,module=jet,excludedTargets={}], v=5140
bbb: [unit=ms,metric=operation.invocations.invocationScanPeriodMillis,excludedTargets={}], v=1000
bbb: [unit=count,metric=operation.responses.backupCount,excludedTargets={}], v=0
bbb: [partitionId=8,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=45
bbb: [unit=count,metric=operation.runningGenericCount,excludedTargets={}], v=0
bbb: [unit=count,metric=raft.metadata.activeMembersCommitIndex,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.asyncOperations,excludedTargets={}], v=0
bbb: [metric=os.totalSwapSpaceSize,excludedTargets={}], v=8589930496
bbb: [unit=count,metric=jobs.completedWithFailure,module=jet,excludedTargets={}], v=3
bbb: [unit=ms,metric=gc.minorTime,excludedTargets={}], v=25429
bbb: [partitionId=7,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=40
bbb: [service=hz:core:clusterService,unit=count,metric=event.publicationCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.generic-operation.thread-0,unit=count,metric=operation.thread.completedRunnableCount,excludedTargets={}], v=0
bbb: [unit=count,metric=event.eventQueueSize,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.callTimeoutCount,excludedTargets={}], v=0
bbb: [genericId=0,unit=count,metric=operation.generic.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=0
bbb: [unit=boolean,metric=partitions.migrationActive,excludedTargets={}], v=1
bbb: [unit=count,metric=jobs.completedSuccessfully,module=jet,excludedTargets={}], v=0
bbb: [unit=count,metric=client.endpoint.totalRegistrations,excludedTargets={}], v=0
bbb: [metric=runtime.freeMemory,excludedTargets={}], v=7169425656
bbb: [unit=ms,metric=cluster.heartbeat.lastHeartbeat,excludedTargets={}], v=0
bbb: [unit=count,metric=blockingWorkerCount,module=jet,excludedTargets={}], v=0
bbb: [partitionId=2,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=40
bbb: [unit=count,metric=operation.invocations.delayedExecutionCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.priority-generic-operation.thread-0,unit=count,metric=operation.thread.completedTotalCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.completedOperationBatchCount,excludedTargets={}], v=35
bbb: [unit=count,metric=operation.responses.missingCount,excludedTargets={}], v=0
bbb: [unit=count,metric=event.syncDeliveryFailureCount,excludedTargets={}], v=0
bbb: [unit=ms,metric=cluster.clock.clusterTimeDiff,excludedTargets={}], v=0
bbb: [unit=count,metric=raft.metadata.groups,excludedTargets={}], v=0
bbb: [unit=count,metric=proxy.createdCount,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.retryCount,excludedTargets={}], v=0
bbb: [partitionId=1,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=40
bbb: [unit=ms,metric=operation.invocations.heartbeatBroadcastPeriodMillis,excludedTargets={}], v=15000
bbb: [unit=bytes,metric=memory.maxNative,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.priority-generic-operation.thread-0,unit=count,metric=operation.thread.errorCount,excludedTargets={}], v=0
bbb: [cooperativeWorker=1,unit=count,metric=taskletCount,module=jet,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.totalPhysical,excludedTargets={}], v=405609000960
bbb: [genericId=2,unit=count,metric=operation.generic.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.completedPacketCount,excludedTargets={}], v=0
bbb: [metric=thread.totalStartedThreadCount,excludedTargets={}], v=382313
bbb: [service=hz:core:proxyService,unit=count,metric=event.publicationCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.generic-operation.thread-0,unit=count,metric=operation.thread.completedOperationCount,excludedTargets={}], v=0
bbb: [unit=ms,metric=gc.unknownTime,excludedTargets={}], v=0
bbb: [partitionId=4,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=56
bbb: [thread=hz.compassionate_brown.generic-operation.thread-1,unit=count,metric=operation.thread.completedRunnableCount,excludedTargets={}], v=0
bbb: [unit=ms,metric=cluster.clock.clusterUpTime,excludedTargets={}], v=3216
bbb: [unit=count,metric=operation.queueSize,excludedTargets={}], v=0
bbb: [genericId=1,unit=count,metric=operation.generic.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=0
bbb: [service=hz:core:clusterService,unit=count,metric=event.listenerCount,excludedTargets={}], v=1
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.completedPacketCount,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.partitionThreadCount,excludedTargets={}], v=2
bbb: [metric=thread.threadCount,excludedTargets={}], v=86
bbb: [unit=count,metric=operation.invocations.heartbeatPacketsSent,excludedTargets={}], v=1
bbb: [unit=count,metric=partitions.replicaSyncRequestsCounter,excludedTargets={}], v=0
bbb: [unit=count,metric=partitions.partitionCount,excludedTargets={}], v=11
bbb: [partitionId=3,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=40
bbb: [metric=runtime.maxMemory,excludedTargets={}], v=8478261248
bbb: [unit=count,metric=operation.invocations.heartbeatPacketsReceived,excludedTargets={}], v=1
bbb: [service=hz:core:partitionService,unit=count,metric=event.publicationCount,excludedTargets={}], v=0
bbb: [metric=thread.daemonThreadCount,excludedTargets={}], v=45
bbb: [thread=hz.compassionate_brown.generic-operation.thread-0,unit=count,metric=operation.thread.completedPartitionSpecificRunnableCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.errorCount,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.freeNative,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.generic-operation.thread-1,unit=count,metric=operation.thread.completedTotalCount,excludedTargets={}], v=0
bbb: [unit=count,metric=cluster.size,excludedTargets={}], v=1
bbb: [unit=count,metric=event.rejectedCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.completedTotalCount,excludedTargets={}], v=396
bbb: [thread=hz.compassionate_brown.generic-operation.thread-1,unit=count,metric=operation.thread.errorCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.completedPartitionSpecificRunnableCount,excludedTargets={}], v=0
bbb: [unit=count,metric=raft.missingMembers,excludedTargets={}], v=0
bbb: [unit=count,metric=jobs.executionCompleted,module=jet,excludedTargets={}], v=5
bbb: [thread=hz.compassionate_brown.generic-operation.thread-1,unit=count,metric=operation.thread.completedPartitionSpecificRunnableCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.errorCount,excludedTargets={}], v=0
bbb: [cooperativeWorker=0,unit=count,metric=taskletCount,module=jet,excludedTargets={}], v=0
bbb: [unit=ms,metric=cluster.clock.localClockTime,excludedTargets={}], v=1669946065103
bbb: [dir=user.home,unit=bytes,metric=file.partition.freeSpace,excludedTargets={}], v=11355586560
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.completedTotalCount,excludedTargets={}], v=263
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.priorityPendingCount,excludedTargets={}], v=0
bbb: [unit=count,metric=gc.majorCount,excludedTargets={}], v=7
bbb: [metric=thread.peakThreadCount,excludedTargets={}], v=1495
bbb: [unit=count,metric=partitions.maxBackupCount,excludedTargets={}], v=0
bbb: [unit=ms,metric=operation.invocations.invocationTimeoutMillis,excludedTargets={}], v=60000
bbb: [thread=hz.compassionate_brown.generic-operation.thread-0,unit=count,metric=operation.thread.errorCount,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.priorityQueueSize,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.completedPartitionSpecificRunnableCount,excludedTargets={}], v=0
bbb: [metric=runtime.totalMemory,excludedTargets={}], v=8478261248
bbb: [unit=ms,metric=operation.invocations.backupTimeoutMillis,excludedTargets={}], v=5000
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.completedOperationBatchCount,excludedTargets={}], v=35
bbb: [unit=count,metric=operation.responses.errorCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.priorityPendingCount,excludedTargets={}], v=0
bbb: [service=hz:core:proxyService,unit=count,metric=event.listenerCount,excludedTargets={}], v=1
bbb: [thread=hz.compassionate_brown.priority-generic-operation.thread-0,unit=count,metric=operation.thread.completedOperationCount,excludedTargets={}], v=0
bbb: [unit=count,metric=proxy.proxyCount,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.freePhysical,excludedTargets={}], v=143138627584
bbb: [partitionId=0,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=40
bbb: [unit=count,metric=operation.responses.timeoutCount,excludedTargets={}], v=0
bbb: [unit=ns,metric=partitions.totalElapsedDestinationCommitTime,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.committedNative,excludedTargets={}], v=0
bbb: [metric=os.openFileDescriptorCount,excludedTargets={}], v=347
bbb: [unit=count,metric=operation.invocations.backupTimeouts,excludedTargets={}], v=0
bbb: [unit=ns,metric=partitions.elapsedMigrationOperationTime,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.genericPriorityQueueSize,excludedTargets={}], v=0
bbb: [unit=count,metric=gc.minorCount,excludedTargets={}], v=195
bbb: [unit=count,metric=transactions.startCount,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.usedMetadata,excludedTargets={}], v=0
bbb: [metric=os.freeSwapSpaceSize,excludedTargets={}], v=6850932736
bbb: [metric=os.totalPhysicalMemorySize,excludedTargets={}], v=405609000960
bbb: [unit=ms,metric=partitions.lastRepartitionTime,excludedTargets={}], v=0
bbb: [unit=count,metric=jobs.executionStarted,module=jet,excludedTargets={}], v=6
bbb: [metric=runtime.availableProcessors,excludedTargets={}], v=8
bbb: [unit=count,metric=event.eventsProcessed,excludedTargets={}], v=21
bbb: [partitionId=10,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=91
bbb: [unit=count,metric=operation.parker.parkQueueCount,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.responses.normalCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.priority-generic-operation.thread-0,unit=count,metric=operation.thread.completedPartitionSpecificRunnableCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.generic-operation.thread-0,unit=count,metric=operation.thread.completedPacketCount,excludedTargets={}], v=0
bbb: [unit=count,metric=client.endpoint.count,excludedTargets={}], v=0
bbb: [unit=count,metric=gc.unknownCount,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.responseQueueSize,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.generic-operation.thread-0,unit=count,metric=operation.thread.completedOperationBatchCount,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.maxMetadata,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.parker.totalParkedOperationCount,excludedTargets={}], v=0
bbb: [unit=count,metric=partitions.localPartitionCount,excludedTargets={}], v=11
bbb: [unit=ms,metric=gc.majorTime,excludedTargets={}], v=3837
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.normalPendingCount,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.runningCount,excludedTargets={}], v=0
bbb: [metric=classloading.loadedClassesCount,excludedTargets={}], v=47701
bbb: [unit=count,metric=operation.invocations.lastCallId,excludedTargets={}], v=293
bbb: [thread=hz.compassionate_brown.generic-operation.thread-1,unit=count,metric=operation.thread.completedOperationBatchCount,excludedTargets={}], v=0
bbb: [unit=count,metric=transactions.commitCount,excludedTargets={}], v=0
bbb: [unit=count,metric=raft.destroyedGroupIds,excludedTargets={}], v=0
bbb: [unit=count,metric=partitions.activePartitionCount,excludedTargets={}], v=11
bbb: [metric=os.maxFileDescriptorCount,excludedTargets={}], v=1048576
bbb: [partitionId=9,unit=count,metric=operation.partition.executedOperationsCount,excludedTargets={MANAGEMENT_CENTER}], v=40
bbb: [service=hz:core:partitionService,unit=count,metric=event.listenerCount,excludedTargets={}], v=3
bbb: [unit=count,metric=operation.invocations.pending,excludedTargets={}], v=2
bbb: [unit=bytes,metric=memory.usedNative,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.completedRunnableCount,excludedTargets={}], v=15
bbb: [dir=user.home,unit=bytes,metric=file.partition.totalSpace,excludedTargets={}], v=21449408512
bbb: [unit=ms,metric=cluster.clock.clusterStartTime,excludedTargets={}], v=1669946061886
bbb: [unit=ms,metric=cluster.clock.clusterTime,excludedTargets={}], v=1669946065109
bbb: [unit=count,metric=partitions.plannedMigrations,excludedTargets={}], v=0
bbb: [dir=user.home,unit=bytes,metric=file.partition.usableSpace,excludedTargets={}], v=11355586560
bbb: [unit=count,metric=proxy.destroyedCount,excludedTargets={}], v=8
bbb: [unit=count,metric=operation.invocations.normalTimeouts,excludedTargets={}], v=0
bbb: [unit=count,metric=raft.metadata.activeMembers,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.failedBackups,excludedTargets={}], v=0
bbb: [unit=count,metric=partitions.memberGroupsSize,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.generic-operation.thread-1,unit=count,metric=operation.thread.completedPacketCount,excludedTargets={}], v=0
bbb: [unit=count,metric=operation.operationTimeoutCount,excludedTargets={}], v=0
bbb: [unit=count,metric=transactions.rollbackCount,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-0,unit=count,metric=operation.thread.completedRunnableCount,excludedTargets={}], v=18
bbb: [thread=hz.compassionate_brown.generic-operation.thread-0,unit=count,metric=operation.thread.completedTotalCount,excludedTargets={}], v=0
bbb: [metric=runtime.uptime,excludedTargets={}], v=2320045
bbb: [unit=count,metric=event.queueCapacity,excludedTargets={}], v=1000000
bbb: [unit=count,metric=raft.nodes,excludedTargets={}], v=0
bbb: [unit=count,metric=partitions.stateStamp,excludedTargets={}], v=-6236066744477238580
bbb: [unit=count,metric=partitions.completedMigrations,excludedTargets={}], v=0
bbb: [thread=hz.compassionate_brown.partition-operation.thread-1,unit=count,metric=operation.thread.normalPendingCount,excludedTargets={}], v=0
bbb: [unit=bytes,metric=memory.usedHeap,excludedTargets={}], v=1309580328
bbb: [name=__jet.executionRecords,unit=ms,metric=map.totalRemoveLatency,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=ms,metric=map.totalPutLatency,excludedTargets={}], v=1
bbb: [name=__jet.executionRecords,unit=ms,metric=map.totalSetLatency,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=ms,metric=map.totalGetLatency,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=ms,metric=map.totalMaxRemoveLatency,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=ms,metric=map.totalMaxGetLatency,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=ms,metric=map.totalMaxSetLatency,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=ms,metric=map.totalMaxPutLatency,excludedTargets={}], v=1
bbb: [name=__jet.executionRecords,unit=ms,metric=map.lastAccessTime,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=ms,metric=map.lastUpdateTime,excludedTargets={}], v=1669946065063
bbb: [name=__jet.executionRecords,unit=count,metric=map.hits,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.numberOfOtherOperations,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.numberOfEvents,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.getCount,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.putCount,excludedTargets={}], v=2
bbb: [name=__jet.executionRecords,unit=count,metric=map.setCount,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.removeCount,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=ms,metric=map.creationTime,excludedTargets={}], v=1669946064581
bbb: [name=__jet.executionRecords,unit=count,metric=map.ownedEntryCount,excludedTargets={}], v=1
bbb: [name=__jet.executionRecords,unit=count,metric=map.backupEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=bytes,metric=map.ownedEntryMemoryCost,excludedTargets={}], v=198
bbb: [name=__jet.executionRecords,unit=bytes,metric=map.backupEntryMemoryCost,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.heapCost,excludedTargets={}], v=198
bbb: [name=__jet.executionRecords,unit=count,metric=map.merkleTreesCost,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.lockedEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.dirtyEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.backupCount,excludedTargets={}], v=1
bbb: [name=__jet.executionRecords,unit=count,metric=map.queryCount,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,unit=count,metric=map.indexedQueryCount,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.totalRemoveLatency,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.totalPutLatency,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.totalSetLatency,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.totalGetLatency,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.totalMaxRemoveLatency,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.totalMaxGetLatency,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.totalMaxSetLatency,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.totalMaxPutLatency,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.lastAccessTime,excludedTargets={}], v=1669946065040
bbb: [name=__jet.results,unit=ms,metric=map.lastUpdateTime,excludedTargets={}], v=1669946065037
bbb: [name=__jet.results,unit=count,metric=map.hits,excludedTargets={}], v=13
bbb: [name=__jet.results,unit=count,metric=map.numberOfOtherOperations,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.numberOfEvents,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.getCount,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.putCount,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.setCount,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.removeCount,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=ms,metric=map.creationTime,excludedTargets={}], v=1669946063888
bbb: [name=__jet.results,unit=count,metric=map.ownedEntryCount,excludedTargets={}], v=2
bbb: [name=__jet.results,unit=count,metric=map.backupEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=bytes,metric=map.ownedEntryMemoryCost,excludedTargets={}], v=2832
bbb: [name=__jet.results,unit=bytes,metric=map.backupEntryMemoryCost,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.heapCost,excludedTargets={}], v=2832
bbb: [name=__jet.results,unit=count,metric=map.merkleTreesCost,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.lockedEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.dirtyEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.results,unit=count,metric=map.backupCount,excludedTargets={}], v=1
bbb: [name=__jet.results,unit=count,metric=map.queryCount,excludedTargets={}], v=4
bbb: [name=__jet.results,unit=count,metric=map.indexedQueryCount,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.totalRemoveLatency,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.totalPutLatency,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.totalSetLatency,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.totalGetLatency,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.totalMaxRemoveLatency,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.totalMaxGetLatency,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.totalMaxSetLatency,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.totalMaxPutLatency,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.lastAccessTime,excludedTargets={}], v=1669946065071
bbb: [name=__jet.records,unit=ms,metric=map.lastUpdateTime,excludedTargets={}], v=1669946065061
bbb: [name=__jet.records,unit=count,metric=map.hits,excludedTargets={}], v=4
bbb: [name=__jet.records,unit=count,metric=map.numberOfOtherOperations,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.numberOfEvents,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.getCount,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.putCount,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.setCount,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.removeCount,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=ms,metric=map.creationTime,excludedTargets={}], v=1669946063888
bbb: [name=__jet.records,unit=count,metric=map.ownedEntryCount,excludedTargets={}], v=1
bbb: [name=__jet.records,unit=count,metric=map.backupEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=bytes,metric=map.ownedEntryMemoryCost,excludedTargets={}], v=5344
bbb: [name=__jet.records,unit=bytes,metric=map.backupEntryMemoryCost,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.heapCost,excludedTargets={}], v=5344
bbb: [name=__jet.records,unit=count,metric=map.merkleTreesCost,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.lockedEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.dirtyEntryCount,excludedTargets={}], v=0
bbb: [name=__jet.records,unit=count,metric=map.backupCount,excludedTargets={}], v=1
bbb: [name=__jet.records,unit=count,metric=map.queryCount,excludedTargets={}], v=4
bbb: [name=__jet.records,unit=count,metric=map.indexedQueryCount,excludedTargets={}], v=0
bbb: [name=__jet.executionRecords,metric=map.store.offloaded.operations.waitingToBeProcessedCount,excludedTargets={}], v=0
bbb: [name=__jet.results,metric=map.store.offloaded.operations.waitingToBeProcessedCount,excludedTargets={}], v=0
bbb: [name=__jet.records,metric=map.store.offloaded.operations.waitingToBeProcessedCount,excludedTargets={}], v=0
bbb: [unit=ms,metric=executionStartTime,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,excludedTargets={}], v=1669946065100
taken
bbb: [unit=ms,metric=executionCompletionTime,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,excludedTargets={}], v=1669946065099
taken
bbb: [unit=count,metric=snapshotBytes,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,excludedTargets={}], v=0
taken
bbb: [unit=count,metric=snapshotKeys,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,excludedTargets={}], v=0
taken
bbb: [unit=count,metric=emittedCount,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,procType=ConvenientSourceP,proc=0,source=true,ordinal=0,excludedTargets={}], v=5
taken
bbb: [key=0,unit=ms,metric=topObservedWm,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,procType=ConvenientSourceP,proc=0,source=true,excludedTargets={}], v=-9223372036854775808
taken
bbb: [key=0,unit=ms,metric=coalescedWm,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,procType=ConvenientSourceP,proc=0,source=true,excludedTargets={}], v=-9223372036854775808
taken
bbb: [key=0,unit=ms,metric=lastForwardedWm,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,procType=ConvenientSourceP,proc=0,source=true,excludedTargets={}], v=-9223372036854775808
taken
bbb: [key=0,unit=ms,metric=lastForwardedWmLatency,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,procType=ConvenientSourceP,proc=0,source=true,excludedTargets={}], v=0
taken
bbb: [unit=count,metric=queuesSize,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,procType=ConvenientSourceP,proc=0,source=true,excludedTargets={}], v=0
taken
bbb: [unit=count,metric=queuesCapacity,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=items,procType=ConvenientSourceP,proc=0,source=true,excludedTargets={}], v=0
taken
bbb: [unit=count,metric=snapshotBytes,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,excludedTargets={}], v=0
taken
bbb: [unit=count,metric=snapshotKeys,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,excludedTargets={}], v=0
taken
bbb: [unit=count,metric=receivedCount,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,procType=AdaptingProcessor,proc=0,sink=true,ordinal=0,excludedTargets={}], v=5
taken
bbb: [unit=count,metric=receivedBatches,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,procType=AdaptingProcessor,proc=0,sink=true,ordinal=0,excludedTargets={}], v=1
taken
bbb: [key=0,unit=ms,metric=topObservedWm,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,procType=AdaptingProcessor,proc=0,sink=true,excludedTargets={}], v=-9223372036854775808
taken
bbb: [key=0,unit=ms,metric=coalescedWm,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,procType=AdaptingProcessor,proc=0,sink=true,excludedTargets={}], v=-9223372036854775808
taken
bbb: [key=0,unit=ms,metric=lastForwardedWm,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,procType=AdaptingProcessor,proc=0,sink=true,excludedTargets={}], v=-9223372036854775808
taken
bbb: [key=0,unit=ms,metric=lastForwardedWmLatency,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,procType=AdaptingProcessor,proc=0,sink=true,excludedTargets={}], v=0
taken
bbb: [unit=count,metric=queuesSize,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,procType=AdaptingProcessor,proc=0,sink=true,excludedTargets={}], v=6
taken
bbb: [unit=count,metric=queuesCapacity,module=jet,job=0908-6189-1108-0001,exec=0908-6189-1109-0001,vertex=loggerSink,procType=AdaptingProcessor,proc=0,sink=true,excludedTargets={}], v=1024
taken
bbb: [name=hz:cluster,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=2
bbb: [name=hz:cluster,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:cluster,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:cluster,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:cluster,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=hz:cluster:event,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=1
bbb: [name=hz:cluster:event,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:cluster:event,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=7
bbb: [name=hz:cluster:event,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:cluster:event,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=jet:coordinator,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=4
bbb: [name=jet:coordinator,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=1
bbb: [name=jet:coordinator,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=142
bbb: [name=jet:coordinator,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=jet:coordinator,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=hz:offloadable,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=8
bbb: [name=hz:offloadable,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:offloadable,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:offloadable,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:offloadable,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=100000
bbb: [name=hz:scheduled:cqc:8322cb90-566c-4f49-b537-b1e630a2f543,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=1
bbb: [name=hz:scheduled:cqc:8322cb90-566c-4f49-b537-b1e630a2f543,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:scheduled:cqc:8322cb90-566c-4f49-b537-b1e630a2f543,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=1
bbb: [name=hz:scheduled:cqc:8322cb90-566c-4f49-b537-b1e630a2f543,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:scheduled:cqc:8322cb90-566c-4f49-b537-b1e630a2f543,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=10000
bbb: [name=hz:client,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=8
bbb: [name=hz:client,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:client,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:client,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:client,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=800000
bbb: [name=hz:cluster:mastership,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=1
bbb: [name=hz:cluster:mastership,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:cluster:mastership,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:cluster:mastership,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:cluster:mastership,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=hz:async,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=8
bbb: [name=hz:async,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=4
bbb: [name=hz:async,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=4
bbb: [name=hz:async,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:async,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=100000
bbb: [name=hz:client-blocking-tasks,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=160
bbb: [name=hz:client-blocking-tasks,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:client-blocking-tasks,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:client-blocking-tasks,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:client-blocking-tasks,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=800000
bbb: [name=hz:mc,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=2
bbb: [name=hz:mc,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:mc,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:mc,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:mc,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2000
bbb: [name=hz:client-query,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=8
bbb: [name=hz:client-query,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:client-query,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:client-query,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:client-query,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=800000
bbb: [name=hz:scheduled,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=16
bbb: [name=hz:scheduled,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:scheduled,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=87
bbb: [name=hz:scheduled,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:scheduled,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=800000
bbb: [name=hz:system,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=8
bbb: [name=hz:system,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:system,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:system,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:system,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=MetricsPublisher,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=16
bbb: [name=MetricsPublisher,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=MetricsPublisher,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=3
bbb: [name=MetricsPublisher,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=MetricsPublisher,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=hz:cluster:splitbrain,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=2
bbb: [name=hz:cluster:splitbrain,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:cluster:splitbrain,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:cluster:splitbrain,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:cluster:splitbrain,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=hz:cluster:version:auto:upgrade,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=1
bbb: [name=hz:cluster:version:auto:upgrade,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:cluster:version:auto:upgrade,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:cluster:version:auto:upgrade,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:cluster:version:auto:upgrade,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=jet:tasklet_initClose,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=8
bbb: [name=jet:tasklet_initClose,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=jet:tasklet_initClose,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=17
bbb: [name=jet:tasklet_initClose,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=jet:tasklet_initClose,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=hz:jet-job-offloadable,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=8
bbb: [name=hz:jet-job-offloadable,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:jet-job-offloadable,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=13
bbb: [name=hz:jet-job-offloadable,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:jet-job-offloadable,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=100000
bbb: [name=hz:query,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=16
bbb: [name=hz:query,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:query,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=0
bbb: [name=hz:query,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:query,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
bbb: [name=hz:CRDTReplicationMigration,unit=count,metric=executor.internal.maximumPoolSize,excludedTargets={}], v=16
bbb: [name=hz:CRDTReplicationMigration,unit=count,metric=executor.internal.poolSize,excludedTargets={}], v=0
bbb: [name=hz:CRDTReplicationMigration,unit=count,metric=executor.internal.completedTasks,excludedTargets={}], v=4
bbb: [name=hz:CRDTReplicationMigration,unit=count,metric=executor.internal.queueSize,excludedTargets={}], v=0
bbb: [name=hz:CRDTReplicationMigration,unit=count,metric=executor.internal.remainingQueueCapacity,excludedTargets={}], v=2147483647
01:54:25,115 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-496 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Completed execution of job '0908-6189-1108-0001', execution 0908-6189-1109-0001
01:54:25,115 DEBUG || - [JobExecutionService] hz.compassionate_brown.cached.thread-8 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Execution of job '0908-6189-1108-0001', execution 0908-6189-1109-0001 completed
01:54:25,116 DEBUG || - [MasterJobContext] ForkJoinPool.commonPool-worker-496 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] job '0908-6189-1108-0001', execution 0908-6189-1109-0001 received response to StartExecutionOperation from [127.0.0.1]:5701: [0, 1, 0, 0, 1, -23, 120, 1, 85, -110, -35, 83, -45, 80, 16, -59, 79, 104, 26, 90, 40, -108, 15, -59, 111, 80, -31, 21, -89, 69, -76, -44, 55, 113, -58, 23, -99, -111, 25, -100, -15, -63, -31, -95, 109, 50, -120, -110, 4, -109, -16, 81, -1, 122, 127, -69, -95, 50, -10, 78, -109, 123, -49, 61, -69, 103, -49, 110, 36, -19, 72, -70, -89, 64, 61, -34, 97, -80, -90, 33, -69, 3, -19, -22, -83, -6, -68, -121, -20, -6, -84, 26, -21, -15, -18, 113, -110, -94, -91, -56, -17, -18, -112, 22, -103, -122, -84, -41, -102, 120, 100, 95, 49, -79, -79, 6, 26, -5, 110, 95, 123, -60, 38, 96, -58, 26, -109, 127, 87, 35, 110, 18, 34, 122, -60, 12, -76, 15, 62, 64, -45, -14, -65, -48, -86, -34, 115, 59, -46, -123, 42, -99, 41, -45, -87, -114, 84, 40, -121, -103, -88, 100, -27, -100, -92, 53, 120, 31, -40, 103, -70, 2, -49, 96, -38, -77, -46, 49, -40, 37, 12, 99, 31, -63, 91, -48, -118, -66, -109, 121, 15, -123, 87, -24, -43, -1, -66, 78, -12, 78, 111, -64, 106, 87, 109, -51, 123, 77, 49, -111, -75, -118, -12, 88, -117, 100, -55, -63, -49, 29, -77, -116, -79, -66, 41, 37, -21, -118, 58, -100, 82, 84, 43, -106, -31, 86, -53, -91, 87, 32, 53, -125, -122, 110, 64, 39, 48, 23, -61, 117, 112, 115, 98, -75, 26, 43, -59, -103, 101, -100, 97, 95, -71, 75, 57, -13, 107, -73, -87, -65, 66, -79, -32, 121, -121, 47, -85, -23, 74, -90, 88, 66, -101, 83, 67, 63, 61, -125, -44, 8, -26, -56, 57, 6, 125, 4, -6, 11, 116, -54, -2, -119, -70, 104, -116, 96, 87, -6, -56, 125, -95, 107, -49, 106, 93, -97, 121, 120, -38, -99, -41, 103, 80, -85, 63, -93, 86, -117, -37, 10, -38, -80, 79, 89, 9, 49, -57, -88, 102, -28, 20, -35, -119, -68, -58, 20, 37, -69, -111, -126, -96, 9, 51, -58, -101, 121, -111, 30, -64, 49, -99, -40, 99, -84, 103, -46, -70, 66, -68, -42, -77, -109, 86, -61, 16, 79, 83, 16, -29, 111, 106, 89, -65, -119, 78, -4, 95, -46, 25, -101, -8, -120, 58, -84, -89, 86, -53, -77, 40, -12, 10, -2, 56, 127, 27, 71, 54, 27, -101, -62, -103, -49, 60, -42, -95, 87, 63, -47, 15, 48, -21, -53, -13, 86, -13, -65, 57, -68, 68, -65, -4, -25, -95, 19, 116, 112, 99, 42, 37, 17, 57, 42, -121, -24, -104, 123, -117, -19, -74, 66, 125, 98, 63, -11, -45, 125, 119, 55, -5, -114, -92, -121, 90, -126, -103, 19, -5, -123, 14, -108, -16, -118, -37, 26, -22, -17, 97, -125, -119, 23, -18, 68, 48, 35, -65, -77, 25, 38, -70, -7, 11, 99, 111, 124, -121, 0, 0, 0, 21, 120, 1, -75, -109, 75, 82, -124, 48, 16, -122, 19, 7, -26, -95, -8, -102, 25, -33, -113, 17, 103, -19, 17, -26, 48, 28, 100, -86, 88, -72, -14, 84, -18, 117, -31, -115, -16, -5, 65, -86, -60, 2, 18, 82, -40, 85, 31, 36, 36, -3, 119, 39, 116, -101, 2, 51, -104, -34, 50, 107, 98, 102, 22, 14, 96, 2, 17, -24, -37, 20, 102, 48, -121, 5, 28, -126, 125, -3, 122, 123, -1, -36, 51, 58, -46, 12, -68, -67, 62, 94, -40, -99, 68, 82, -19, -14, 59, 102, -19, 4, -2, 70, -27, 19, -90, -88, -89, 16, -26, -67, -61, -13, -84, 60, 2, -125, -74, -92, -21, -32, -25, -84, 47, 97, 5, 107, -72, -128, 75, -72, -6, -103, -73, 38, 23, -89, -84, 94, -61, -115, 118, 89, 93, -105, 43, -51, -82, 56, 77, -3, 28, 37, -103, 14, 127, 11, -29, -85, -34, -3, -117, -22, -3, 104, -86, 8, -107, -106, -15, 124, -88, 106, -74, 40, 84, -93, -29, 92, 69, 37, 94, 93, -16, -29, -24, -86, -86, -70, -60, 85, -14, 27, 54, 53, -1, -6, -17, -92, 92, 37, -33, -19, -83, -32, 79, -82, -110, -105, -69, 74, 49, -123, -70, -28, -97, 25, -69, 75, 94, 37, -71, -123, -66, -1, 16, -84, 110, -107, -113, 127, 67, -11, -59, 105, 94, 109, -114, -82, -52, -89, -95, -62, 84, 93, 13, 21, -90, -22, 106, 40, 127, -43, -22, -4, -58, 100, 12, -68, 27, 106, -80, -4, 84, 23, -20, 106, -88, -95, -86, -111, -7, 6, -22, -83, 63, 121] @ 1669946065114
01:54:25,117 DEBUG || - [MasterJobContext] ForkJoinPool.commonPool-worker-496 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Execution of job '0908-6189-1108-0001', execution 0908-6189-1109-0001 was successful
01:54:25,118 DEBUG || - [JobClassLoaderService] hz.compassionate_brown.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Finish JobClassLoaders phaseCount = 0, removing classloaders for jobId=0908-6189-1108-0001
01:54:25,120  INFO || - [MasterJobContext] hz.compassionate_brown.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] Execution of job '0908-6189-1108-0001', execution 0908-6189-1109-0001 completed successfully
	Start time: 2022-12-02T01:54:25.062
	Duration: 00:00:00.056
	Vertices:
		items
			emittedCount: 5
		loggerSink
			receivedCount: 5
01:54:25,121 DEBUG || - [JobCoordinationService] hz.compassionate_brown.cached.thread-7 - [127.0.0.1]:5701 [dev] [5.3.0-SNAPSHOT] job '0908-6189-1108-0001', execution 0908-6189-1109-0001 is completed
01:54:25,886  INFO |testExecutionMetricsBatchJob| - [SimpleTestInClusterSupport] Time-limited test - Ditching 3 jobs in SimpleTestInClusterSupport.@After: [0908-6189-1103-0001, 0908-6189-1106-0001, 0908-6189-1108-0001]
01:54:25,888  INFO |testExecutionMetricsBatchJob| - [SimpleTestInClusterSupport] Time-limited test - Destroying 0 distributed objects in SimpleTestInClusterSupport.@After: []
BuildInfo right after testExecutionMetricsBatchJob(com.hazelcast.jet.core.metrics.JobExecutionMetricsTest): BuildInfo{version='5.3.0-SNAPSHOT', build='20221202', buildNumber=20221202, revision=2e15289, enterprise=false, serializationVersion=1}
Hiccups measured while running test 'testExecutionMetricsBatchJob(com.hazelcast.jet.core.metrics.JobExecutionMetricsTest):'
01:54:25, accumulated pauses: 758 ms, max pause: 736 ms, pauses over 1000 ms: 0


No metrics recorded during the test

```
</details>

Standard output can be found here - https://s3.console.aws.amazon.com/s3/buckets/j-artifacts/Hazelcast-master-RedHatJDK8/67/

--- a/hazelcast/src/main/java/com/hazelcast/jet/impl/execution/ExecutionContext.java
+++ b/hazelcast/src/main/java/com/hazelcast/jet/impl/execution/ExecutionContext.java
@@ -229,6 +229,7 @@ public class ExecutionContext implements DynamicMetricsProvider {
                 if (cl == null) {
                     cl = nodeEngine.getConfigClassLoader();
                 }
+                startTime.set(System.currentTimeMillis());
                 executionFuture = taskletExecService
                         .beginExecute(tasklets, cancellationFuture, cl)
                         .whenComplete(withTryCatch(logger, (r, t) -> setCompletionTime()))
@@ -241,7 +242,6 @@ public class ExecutionContext implements DynamicMetricsProvider {
                             }
                             return res;
                         });
-                startTime.set(System.currentTimeMillis());
             }
             return executionFuture;
         }

